src/ceph-volume/ceph_volume/tests/devices/lvm/test_batch.py

```diff
@@ -1,11 +1,61 @@
+import mock
 from ceph_volume.devices.lvm import batch
 
 
-class TestBatchSmoke(object):
+class TestBatch(object):
 
-    def test_batch_instance(self, is_root):
-        b = batch.Batch([])
-        b.main()
+	def test_batch_instance(self, is_root):
+		b = batch.Batch([])
+		b.main()
+
+	@mock.patch('ceph_volume.devices.lvm.batch.disk')
```

Comment:
this test looks OK, if you could update it with the monkeypatch fixture we are good to move forward I think